### ISSUE TITLE
docker: KES binary does not report expected version number

### ISSUE THREAD

**Issue by PhilippeAccorsi** [minio/kes#100]
#### Bug describtion

We test Minio with kes server and that working good. Now we tried to use API for monitoring (and seen what can be done with API) and we seen a wrong version number when we execute this command curl --key metric.key --cert metric.cert --request GET 'https://kes:7373/version' -k from client (kes is set in /etc/hosts to use my VM where KES serveur running):
~~~
{"version":"0.0.0-dev"}
~~~

- We use docker images `minio/kes:v0.13.0`.

#### Expected behavior

API request for `/version` show the real version running

#### Additional context

We use Minio and KES server with docker images.

**Comment by aead** [comment]
@PhilippeAccorsi What version does the docker container print when you run e.g. `docker run <image-id> --version`
Does it show `v0.13.0` or `0.0.0-dev`?

**Comment by PhilippeAccorsi** [comment]
Thanks for your answer and your investiguation :)

**Comment by aead** [comment]
#101

**Comment by aead** [comment]
Closing this as incorrect fix as discussed with @harshavardhana

**Comment by aead** [comment]
@PhilippeAccorsi Can you try the patch release `v0.13.1` - i.e:
```
docker run minio/kes:v0.13.1 --version
```
It seems like this issue is already resolved.

**Comment by PhilippeAccorsi** [comment]
> @PhilippeAccorsi Can you try the patch release `v0.13.1` - i.e:
> 
> ```
> docker run minio/kes:v0.13.1 --version
> ```
> 
> It seems like this issue is already resolved.

Yes this is working good :+1: 

Thanks for your answer :)

**Comment by PhilippeAccorsi** [comment]
We just have tested the v0.13.0, but I think we need to test the last version before deploying this on production.

**Comment by PhilippeAccorsi** [comment]
I close this issue because fix is already done since v0.13.1
